File: Assets/PlayerScript.js
#pragma strict

var up : KeyCode;
var down : KeyCode;

var speed : float = 10;






function Start () {

}

function Update () {

	//if(Input.GetKey(up))
	if(Input.GetAxis("Mouse Y") > 0)
	{
		rigidbody2D.velocity.y = speed;
	}
	//else if(Input.GetKey(down))
	else if(Input.GetAxis("Mouse Y") < 0)
	{
		rigidbody2D.velocity.y = speed *-1;
	}
	else
	{
		rigidbody2D.velocity.y = 0;
	}
	
	rigidbody2D.velocity.x = 0;
		

}

File: Assets/PlayerScript.cs
using UnityEngine;
using System.Collections;

public class PlayerScript : MonoBehaviour {

	public float paddleSpeed = 15f;

	public float range = 5f;




	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {

		float input = Input.GetAxis("Vertical");

		// move the paddle
		Vector2 pos = transform.position;
		pos.y += input * paddleSpeed * Time.deltaTime;

		pos.y = Mathf.Clamp(pos.y, -range, range);

		transform.position = pos;
	
	}


}
